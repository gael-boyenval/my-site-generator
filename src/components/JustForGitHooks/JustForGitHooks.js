import React from 'react';
import styled from 'styled-components';

const X = styled.div``;

const JustForGitHooks = () => <X>hello</X>;

export default JustForGitHooks;
