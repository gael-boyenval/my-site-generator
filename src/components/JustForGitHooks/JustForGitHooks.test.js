/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import 'jest-styled-components';

import JustForGitHooks from './JustForGitHooks';

configure({ adapter: new Adapter() });

describe('Component JustForGitHooks', () => {
  it('should be render without crash', () => {
    const component = renderer.create(<JustForGitHooks />);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
