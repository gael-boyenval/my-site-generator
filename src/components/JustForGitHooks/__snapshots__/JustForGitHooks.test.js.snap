// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component JustForGitHooks should be render without crash 1`] = `
<div
  className=""
>
  hello
</div>
`;
